### DISCUSSION TITLE
Wondering if you are waiting for 4.0 to be fully released before you release Pi versions?

### DISCUSSION THREAD

**Discussion by fidgetbytes** [hiulit/Unofficial-Godot-Engine-Raspberry-Pi#26] [General]
Just wondering if you are going to be releasing any of the beta versions for people to mess with, or are you waiting until the full release?

**Comment by hiulit** [comment]
Hi there!
I've already tried compiling the latest betas of Godot 4 on the Pi 4, and the compilation works fine but then it doesn't launch, because errors :P
We might have to wait until 4.1 because I think the current video driver support is only Vulkan and GLES3 and neither of those work with the Pi.
There are Vulkan drivers for the Pi, and I have them installed, but they might not be compliant with the Godot standards.
